src/ImageSharp/Processing/Processors/Convolution/Convolution2PassProcessor{TPixel}.cs

```diff
@@ -396,6 +396,8 @@ private void Convolve4(int y, Span<Vector4> span)
 
                     PixelOperations<TPixel>.Instance.ToVector4(this.configuration, sourceRow, sourceBuffer);
 
+                    Numerics.Premultiply(sourceBuffer);
```

Comment:
This can be made conditional on `PixelTypeInfo.AlphaRepresentation`:

https://github.com/SixLabors/ImageSharp/blob/9777a3b84eb8a38adb09c13577be393980116b28/src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeProcessor%7BTPixel%7D.cs#L191-L194

Reply:
Yeah. Something to look into. I keep meaning to see what else we can add to `PixelTypeInfo` actually that would help us in other places.